external-adapter/src/config.ts

```diff
@@ -5,26 +5,30 @@ import { logger } from './logger'
 const ENV_API_KEY = 'API_KEY'
 const ENV_API_ENDPOINT = 'API_ENDPOINT'
 const ENV_API_TIMEOUT = 'API_TIMEOUT'
+const ENV_API_VERBOSE = 'API_VERBOSE'
 const DEFAULT_API_TIMEOUT = 30000
 
 export const constants = {
   ENV_API_KEY,
   ENV_API_ENDPOINT,
   ENV_API_TIMEOUT,
   DEFAULT_API_TIMEOUT,
+  ENV_API_VERBOSE,
 }
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 const cloneNoSecrets = (config: Config): Config => (({ apiKey, ...o }) => o)(config)
 
 export function getDefaultConfig(prefix = ''): Config {
   const apiKey = util.getEnv(ENV_API_KEY, prefix)
+  const timeout = util.getEnv(ENV_API_TIMEOUT, prefix)
   return {
     apiKey,
+    verbose: !!util.getEnv(ENV_API_VERBOSE, prefix),
     api: {
       withCredentials: !!apiKey,
       baseURL: util.getEnv(ENV_API_ENDPOINT, prefix),
-      timeout: parseInt(util.getEnv(ENV_API_TIMEOUT, prefix)) || DEFAULT_API_TIMEOUT,
+      timeout: timeout ? parseInt(timeout) : DEFAULT_API_TIMEOUT,
```

Comment:
`parseInt` will just return a `NaN` on parse error, so we can just do:

```suggestion
      timeout: parseInt(timeout) || DEFAULT_API_TIMEOUT,
```